gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Rename bug-report.md to bug-template.md

### Description

Our bug template is not given as an option when making a new issue.
![image](https://user-images.githubusercontent.com/13837978/69604839-908bc900-0fec-11ea-90bf-e04eb4840b3f.png)

My guess is that it's because the file's name is "bug-report.md" instead of "bug-template.md", which is what [Dan's instructions](https://docs.google.com/document/d/1tE9jQ3AMKzqNTT3zUYk9J-ZIZej9JNcWzaD7BAywc_o/edit) say to name it.

### Testing Directions

1. I don't know of any way to test this but I see that other teams have "bug-template.md" and when I try to make an issue in their repos, I'm given the option to use their bug template

### Miscellaneous Comments

* We should get this bug template working for tomorrow's testing day
